app/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,8 @@
     <string name="settings_backup_status_summary">Last backup: %1$s</string>
     <string name="settings_backup_exclude_apps">Exclude apps</string>
     <string name="settings_backup_now">Backup now</string>
+    <string name="settings_backup_recovery_code">Recovery Code</string>
```

Comment:
This isn't related to this commit, but I think this is one the things that should be renamed.

Backup code / Backup passphrase / Backup key all sound better / clearer than Recovery code.

Can be it's own commit / PR, just bringing it up now since we're adding this new screen.

Reply:
Not sure I agree here. This is something for UX research.